proxy-node-shared/src/main/java/uk/gov/ida/notification/validations/Base64SamlValidator.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.ida.notification.validations;
+
+import org.apache.commons.lang3.StringUtils;
+import org.glassfish.jersey.internal.util.Base64;
+import org.opensaml.saml.saml2.core.impl.ResponseImpl;
+import uk.gov.ida.notification.saml.SamlParser;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class Base64SamlValidator implements ConstraintValidator<ValidBase64Saml, String> {
+
+    public static final int MinLength = 64;
+    public static final int MaxLength = 64000;
+
+    @Override
+    public void initialize(ValidBase64Saml constraint) { /* intentionally blank */ }
+
+    @Override
+    public boolean isValid(String value, ConstraintValidatorContext context) {
+        if (StringUtils.isBlank(value)) { return true; } // @NotNull should detect nulls
```

Comment:
The comment at the end is redundant. Is the expected value of value to be a base64 encoded string?
Even an empty base64 string will have a length of 4.

Reply:
The value could be null or empty or whitespace, so the comment is to indicate that we're accepting empty values in this validator, with the expectation that a developer will use @NotNull if they don't allow nulls.